jwst/wfs_combine/wfs_combine_step.py

```diff
@@ -60,7 +65,7 @@ def process(self, input_table):
             # Save the output file
             if self.save_results:
                 self.save_model(
-                    output_model, suffix='wfscmb', output_file=outfile, format=False
+                    output_model, suffix='wfscmb.fits', output_file=outfile, format=False
```

Comment:
I don't think `fits` should be included in the specification of the file name suffix. `suffix` is intended to specify only the final suffix contained in the root part of the file name. The file name extension is set within either the datamodels methods or stpipe based on the file format used (FITS in this case).

Reply:
removed ".fits" from name.

Reply:
Actually, `self.suffix` should be defined for the class, and then it will work as intended.  This can be done in the `spec` class attribute so that there's a default, but that it can be overridden if needed.

Reply:
James, can you point me to an example of some code doing this? I've never really groked the pipeline class structure.

Reply:
Here's how its done in `SourceCatalogStep`.  You can do the same here I think.

https://github.com/spacetelescope/jwst/blob/fc8a4077904d27898a8090ff467634c36435565a/jwst/source_catalog/source_catalog_step.py#L41

Reply:
Is this code section even supposed to be here? I can't find the equivalent in any other step. When I try to run the step setting save_result to True, it crashes with an incorrect file extension exception. 
The code after this sets save_results to False. So, I'm really confused. The comments claim that Format=True needs to be set. I don't know what that does but something is wrong,.